providers/curation/github.js

```diff
@@ -304,19 +326,21 @@ class GitHubCurationService {
     return coordinates.toString()
   }
 
-  _getBranchName(coordinates) {
-    const c = coordinates
-    return `${c.type.toLowerCase()}_${c.name}_${c.revision}_${moment().format('YYMMDD_HHmmss.SSS')}`
+  async _getBranchName(userGithub, serviceGithub, info) {
+    const committer = userGithub ? info : await serviceGithub.users.get({})
```

Comment:
should not have to do this now that we have the userInfo cached in the request. Might have to pass in the userInfo rather than the two clients.